grakn-core/src/main/java/ai/grakn/graql/Pattern.java

```diff
@@ -42,4 +42,16 @@
      */
     @CheckReturnValue
     PatternAdmin admin();
+
+    /**
+     * Join patterns in a conjunction
+     */
+    @CheckReturnValue
+    Pattern and(Pattern pattern);
```

Comment:
Why did quickcheck refactors need these methods to be added to the `Pattern` interface?